reana_commons/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.9.4a2"
+__version__ = "0.9.4a4"
```

Comment:
Let's regenerate the release commit so that the new version is 0.9.4a3 and not 0.9.4a4